src/stringGen.js

```diff
@@ -0,0 +1,20 @@
+import fs from 'fs';
+
+const playerNum = 2;
+
+const baseArray = ['a', 'a'];
+const idArray = baseArray.map(() => makeid());
+
+fs.writeFileSync('regIDArray.json', JSON.stringify(idArray, null, 2));
+
+
+function makeid()
+{
+    let text = "";
+    const possible = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789";
+
+    for( let i=0; i < 5; i++ )
+        text += possible.charAt(Math.floor(Math.random() * possible.length));
```

Comment:
I find the standard random function a bitch to work with sometimes.  My preference would be to find a random library that lets me do something like:  `random.choose(possible, 5);`